docs-ref-conceptual/includes/cli-install-linux-apt.md

```diff
@@ -40,25 +40,28 @@ If you prefer a step-by-step installation process, complete the following steps
 
     ```bash
     sudo apt-get update
-    sudo apt-get install ca-certificates curl apt-transport-https lsb-release gnupg
+    sudo apt-get install apt-transport-https ca-certificates curl gnupg lsb-release
     ```
 
 2. Download and install the Microsoft signing key:
 
     ```bash
     sudo mkdir -p /etc/apt/keyrings
     curl -sLS https://packages.microsoft.com/keys/microsoft.asc |
-        gpg --dearmor |
-        sudo tee /etc/apt/keyrings/microsoft.gpg > /dev/null
+      sudo gpg --dearmor -o /etc/apt/keyrings/microsoft.gpg
```

Comment:
This causes unexpected behavior change. See https://github.com/MicrosoftDocs/azure-docs-cli/pull/4417.